### ISSUE TITLE
Brainstorm a way to get people to submit their own experiences

### ISSUE THREAD

**Issue by mayawagon** [bklynlibrary/bklynConnect-curriculum#10]
We could have commenting, but I think it makes sense for people to submit more formally if this is something they're actually running?

What if there was a form that said something like "tell us about your group and check each module that you used in your curriculum. How did it work? What could have gone better? What questions did people have?" then manually add it to the site

**Comment by mayawagon** [comment]
How can we get students-teachers and teacher-students to submit experiences? As well as other people in the surrounding community who are involved?

**Comment by ghost** [comment]
Experiences on what?

**Comment by mayawagon** [comment]
Hi Mary Anna! Experiences on using the curriculum in your own context. Questions like: What happened? Did it go well? Would you change anything?

**Comment by ucanmaketv** [comment]
Surveys are effective tool especially if data analysys is important to project.Develop post course participant survey that course facilitators may use and modify for target audience. Offer free/low cost incentives for feedback such as mentions and monthly giveaway such as t-shirt.

Use social media campaigns to encourage users connect and share during after course.

**Comment by ucanmaketv** [comment]
Found StoryEngine searching Moz Sprint projects http://loupdesign.wpengine.com/about/

**Comment by Claw33** [comment]
Yes to StoryEngine (yes I am biased.) StoryEngine is a project of Loup. [Loup](https://www.loup.design/manifesto/) is a design thinking consultancy that helps organizations listen to their customers, stakeholders and staff — then translate what they’re learning into powerful insights and communications assets. Her are some resources:

- Global Sprint Interview guide:  https://docs.google.com/document/d/1xDY8SnLN3Lm2WOa12KPtxHg9YqG5jMrWGE7m7HOFTpY/edit#heading=h.5jsd47vpovmk

 - Contributing to StoryEngine: https://github.com/LoupDesign/StoryEngine/blob/master/CONTRIBUTING.md

- StoryEngne repo: https://github.com/LoupDesign/StoryEngine

- StoryEngine story database: https://storyengine.io/

- History of project: https://medium.com/@MozOpenLeaders/the-magic-is-in-the-listening-40d065465337

**Comment by prefontaine** [comment]
Hi @mayawagon a few follow up-ideas. 

- Explore having people / groups to interview each other -- that way you highlight folks and connect them while _you're_ learning
- Use quick analysis from first few stories to share highlights and create a series of micro-surveys that can accompany project updates
